style.css

```diff
@@ -0,0 +1,210 @@
+    width: 100%;
+    height: 210px;
+    border: 2px;
+    padding: 0.5px;
+}
+
+.footer-content {
+    text-align: center;
+    width: 100%;
+    display: block;
+    position: absolute;
+    background: white;
+}
+
+.footer-content a {
+    text-decoration: none;
+    color: #348dce;
+}
+
+.footer-content > .spacer {
+    line-height: 12px;
+    padding-left: 10px;
+}
+
+.main_nav {
+    color: white;
+    display: block;
+    position: relative;
+    max-width: 76%;
+    margin: auto;
+    top: 50px;
+    height: 36px;
+    z-index: 9998;
+    border-radius: 4px;
+    background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+}
+
+.main_nav ul {
+    display: inline-block;
+    color: white;
+    list-style: none;
+    height: 36px;
+    padding: 0 !important;
+    margin: 0 !important;
+    text-decoration: #fff;
+}
+
+.main_nav li {
+    display: inline-block;
+    background-color: transparent;
+    color: white;
+    list-style-type: none;
+    padding: 0 !important;
+    margin: 0 !important;
+    width: 106px;
+    height: 36px;
+    border-right: 1px solid rgb(24, 24, 24);
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    background-image: url(/web/20140301004259im_/http://images.apple.com/global/nav/images/globalnav_text.svg);
+}
+
+.main_nav a {
+    display: block;
+    background-color: transparent;
+    text-align: center;
+    height: 36px;
+    padding: 10px 0;
+    width: 100%;
+    color: white;
+    text-decoration: none;
+    -webkit-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+}
+
+.main_nav input {
+    border-radius: 12px;
+    background: transparent;
+    height: 19px;
+    background-image: none;
+    box-sizing: border-box;
+    display: inline;
+}
+
+.main_nav .fa-search {
+    height: 12px;
+    width: 12px;
+    display: inline;
+    position: relative;
+    right: -20px;
+    color: black;
+}
+
+.main_nav .fa-apple {
+    height: 25px;
+    width: 25px;
+}
+
+.main_nav .search-store {
+    width: 80px;
+    display: inline;
+    box-sizing: content-box;
+    padding: 0 20px;
+}
+
+.main_nav input[type="text"] {
+    width: 80px;
+    display: inline;
+    transition-timing-function: ease-in;
+    transition-duration: 10ms;
+    box-sizing: content-box;
+    padding: 0 20px;
+    direction: ltr;
+}
+
```

Comment:
Using different units could help with adapting the website to different screen sizes.

Reply:
I have worked on:-
- Updating some sections with **max-width** has resolved most of the issues (except mobile). Thanks for pointing me to the link
- Added links to the products
- Changed the footer margins and line spacing to look like than the original site

`Let me know if there is anything else to work on`